code/game/machinery/accounting.dm

```diff
@@ -36,8 +36,19 @@
 			bank_account.account_job = /datum/job/unassigned
 		playsound(loc, 'sound/machines/synth_yes.ogg', 30 , TRUE)
 		to_chat(user, span_notice("New account registered under account identification number [bank_account.account_id]."))
+		balloon_alert(user, "new account [bank_account.account_id] registered")
```

Comment:
don't balloon alert the account ID please, just that a new account was registered. Even better, just make the machine say it, and say will handle the balloon alert.